Formula/easyrpg-player.rb

```diff
@@ -21,6 +20,7 @@ class EasyrpgPlayer < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "expat"
```

Comment:
Does
```ruby
uses_from_macos "expat"
```
not work?

Reply:
No. The only way I got them to build locally was to use `depends_on`. If you build `liblcf` with `uses_from_macos` then `easyrpg_player` fails.

Reply:
Probably worth a comment mentioning this in case someone tries to replace the `depends_on` with `uses_from_macos` at some point.

Reply:
Hey. EasyRPG contributor here.

Does macOS bundle the expat library with the system?

That's new to me. In our tests we always build it manually which explains why nobody ever noticed that this is broken. Maybe we can fix this in the next release 🤔

Is there a way for me to reproduce the failure when ``uses_from_macos`` is used?

Reply:
@Ghabry Here is the error you're requesting:
```
CMake Error at /opt/homebrew/Cellar/cmake/3.26.3/share/cmake/Modules/CMakeFindDependencyMacro.cmake:76 (find_package):
  Could not find a package configuration file provided by "expat" with any of
  the following names:

    expatConfig.cmake
    expat-config.cmake

  Add the installation prefix of "expat" to CMAKE_PREFIX_PATH or set
  "expat_DIR" to a directory containing one of the above files.  If "expat"
  provides a separate development package or SDK, be sure it has been
  installed.
Call Stack (most recent call first):
  /opt/homebrew/lib/cmake/liblcf/liblcf-config.cmake:36 (find_dependency)
  builds/cmake/Modules/PlayerFindPackage.cmake:54 (find_package)
  CMakeLists.txt:810 (player_find_package)


-- Configuring incomplete, errors occurred!
```

Reply:
@Ghabry:

> Does macOS bundle the expat library with the system?

Yes, it does. Based on the snippet in the log above, it looks like the EasyRPG build is looking for a CMake config file for expat, but the `libexpat` bundled with macOS does not provide one.